### DISCUSSION TITLE
<Request> - Multi Factor Authentication

### DISCUSSION THREAD

**Discussion by Danielson89** [Freika/dawarich#460] [Ideas]
Hi,

With all the security craziness out there would it be possible to have Google Authenticator as an option.  I'd use this straight away.  Nearly all of my tools at home are using MFA.

Thanks

**Comment by redge76** [comment]
Just use a reverse proxy in front of all your services. It will be a lot more efficient than configuring MFA/authentication for each self-hosted application.
(See Cloudflare Access if you want an easy solution)
What would really help is some king of trusted header authentication so that you can rely on the reverse proxy security for SSO.
